pkg/fanal/artifact/image/remote_sbom.go

```diff
@@ -59,24 +59,32 @@ func (a Artifact) inspectSBOMAttestation(ctx context.Context) (ftypes.ArtifactRe
 	}
 
 	log.Logger.Debugf("Found matching Rekor entries: %s", entryIDs)
-	for _, id := range entryIDs {
-		log.Logger.Debugf("Inspecting Rekor entry: %s", id)
-		ref, err := a.inspectRekorRecord(ctx, client, id)
-		if errors.Is(err, rekor.ErrNoAttestation) || errors.Is(err, errNoSBOMFound) {
-			continue
-		} else if err != nil {
-			return ftypes.ArtifactReference{}, xerrors.Errorf("rekor record inspection error: %w", err)
+
+	for i := 0; i < len(entryIDs); i += rekor.MaxGetEntriesLimit {
+		end := i + rekor.MaxGetEntriesLimit
+		if end > len(entryIDs) {
+			end = len(entryIDs)
+		}
```

Comment:
`Chunk` may help.
https://github.com/samber/lo#chunk

Reply:
Thank you! I have changed to use the lo.Chunk.